runtime/crab/src/pallets/assets.rs

```diff
@@ -24,16 +24,19 @@ pub enum AssetIds {
 	CKton = 1026,
 }
 
+frame_support::parameter_types! {
+	pub Signers: Vec<AccountId> = vec![ROOT];
```

Comment:
O, suggest renaming this, give it a name like `creators`. I did not think too much when writing this example. Sorry for that.